adal/src/main/java/com/microsoft/aad/adal/AuthenticationResult.java

```diff
@@ -440,13 +444,13 @@ final void setFamilyClientId(final String familyClientId) {
         mFamilyClientId = familyClientId;
     }
 
-    final void setAuthority(final String authority) {
+    public final void setAuthority(final String authority) {
         if (!StringExtensions.isNullOrBlank(authority)) {
             mAuthority = authority;
         }
     }
 
-    final CliTelemInfo getCliTelemInfo() {
+    public final CliTelemInfo getCliTelemInfo() {
```

Comment:
Same note as above, is `public` vis required?

Reply:
this method is called in Ad-accounts, so I need to make it public as it was in Unity-fork